x-pack/platform/plugins/shared/stack_connectors/server/connector_types/email/index.ts

```diff
@@ -100,6 +100,15 @@ function validateConfig(
     throw new Error(`[from]: ${invalidEmailsMessage}`);
   }
 
+  const { oauthTokenUrl } = config;
+  if (oauthTokenUrl) {
+    if (!configurationUtilities.isUriAllowed(oauthTokenUrl)) {
```

Comment:
nit
```suggestion
  if (oauthTokenUrl && !configurationUtilities.isUriAllowed(oauthTokenUrl)) {
```

Reply:
fixed in 652f61b26ab4606d52f808672f77f78aa061c617